oeps/oep-0008.rst

```diff
@@ -0,0 +1,323 @@
+==================================
+OEP-8: Unit Testing Best Practices
+==================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-8 </oeps/oep-0008>`             |
++===============+===========================================+
+| Title         | Unit Testing Best Practices               |
```

Comment:
In addition, to @benpatterson's point, there are many practices around unit testing which this OEP won't cover: when do you mock, and what granularity to test, etc.  There are lots of unit testing techniques and approaches that this document isn't meant to discuss.

Perhaps this is less about unit testing, and more about repo structure and tooling?